### DISCUSSION TITLE
NRA?

### DISCUSSION THREAD

**Discussion by Heinrich-Ffm** [Akaflieg-Freiburg/enroute#401] [Ideas]
Naturschutzgebiete? / Vogelschutzgebiete?
Ich würde die gern ausblenden.
Vögel begegnen einem auch außerhalb dieser Gebiete, ich bleibe im Reiseflug eh drüber und schaue raus.

**Comment by kebekus** [comment]
@Heinrich-Ffm Das mit dem Ausblenden ist technisch machbar, aber schwierig. Das Verkaufsargument unserer App war bislang immer die Einfachheit, und das bedeutet auch: möglichst wenige Einstellmöglichkeiten. Es ist klar, dass wir damit nicht jedem Wunsch gerecht werden…

Ich rate auch inhaltlich sehr davon ab, diese Gebiete auszublenden. Es ist zwar in Deutschland im Moment so, dass der Einflug zurzeit nicht mehr bestraft wird. Das ist aber im benachbarten Ausland ganz anders -- in Österreich (z.B. südlich von Zell am See) reichen diese Gebiete in große Höhen, der Einflug kostet schnell einen füfnstelligen (!) Betrag. Ähnlich in Italien südlich von Aosta.
